src/side-navigation/side-navigation.scss

```diff
@@ -99,18 +99,19 @@
       @include themed {
         border-color: t(iui-color-background-5);
       }
-    }
 
-    > .iui-button-icon {
-      width: $iui-icons-large;
-      height: $iui-icons-large;
-      flex-shrink: 0;
+      //boost specificity
+      > .iui-button-icon.iui-button-icon {
```

Comment:
This is now boosting the specificity to 0-6-0 (and the label has 0-7-0), which hardly seems necessary. Why do you need it?